tools/lisa-build-asset

```diff
@@ -164,6 +215,9 @@ def main():
     parser.add_argument('--build-dir', help='Build director. Defaults to temp folder')
     args = parser.parse_args()
 
+    if args.build_dir:
```

Comment:
I usually  avoid modifying the args inplace, so it makes it more obvious what is straight from argparse and what is the result of some processing:
```
build_dir = os.path.abspath(args.build_dir) if args.build_dir else None
```